pyaedt/edb_core/hfss.py

```diff
@@ -663,7 +664,8 @@ def create_edge_port_horizontal(
             Name of the port. The default is ``None``.
         impedance : int, float, optional
             Impedance of the port. The default value is ``50``.
-
+        layer_alignment : str, optional
+            Layer alignment. The default value is ``Upper``.
```

Comment:
Based on the code below, the `layer_alignment` argument can only take two values: `Upper` and `Lower`.
Maybe we could mention it in the description here.